lib/xboxrt/vcruntime/excpt.cpp

```diff
@@ -32,6 +32,23 @@ extern "C" int _nested_unwind_handler (_EXCEPTION_RECORD *pExceptionRecord, EXCE
     return DISPOSITION_COLLIDED_UNWIND;
 }
 
+static inline void *call_ebp_func (void *func, void *_ebp)
+{
+    void *result;
+
+    asm volatile (
+        "pushl %%ebp;"
```

Comment:
Not sure, but there might be SSE stack alignment violations here (I assume this also affects Pentium 3?).
I believe the stack must be incremented and decremented in steps of 0x10 bytes if the callee (potentially) uses local SSE variables (I think gcc started doing this).
Otherwise, if the handler has local SSE variables, those might be suddenly misaligned and then the CPU would crash on access (when using aligned access, which is faster).

We might also violate similar rules elsewhere in nxdk, and I'm not sure about the specifics. I just wanted to mention it in case there are weird (seemingly random) crashes later on. Someone should probably take a look at the compiler docs.

I'm also aware of similar rules with malloc, because some C compilers will assume specific alignments on returned buffers - while working on some game wrappers for Windows games I ran into this, sometimes as late as 20 minutes into gameplay, because some tiny-mem-allocator would double-free due to some address-logic.